ci-templates/gitlab/synchronise_elements.yml

```diff
@@ -2,7 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 variables:
-  CAPELLA2POLARION_DEBUG: "1"
+  CAPELLA2POLARION_DEBUG: 1
+  CAPELLA2POLARION_GROUPED_LINKS_CUSTOM_FIELDS: 1
```

Comment:
This would become redundant when declaring the proper default in the Python code, as described in another comment.